lib/kaiser/cli.rb

```diff
@@ -387,7 +391,11 @@ def db_params
     end
 
     def db_image
-      Config.kaiserfile.database[:image]
+      platform = ''
+      platform = "--platform #{Config.kaiserfile.database[:platform]}" if Config.kaiserfile.database[:platform]&.length
```

Comment:
It can be an empty string. This will result in a fake true

<img width="222" alt="スクリーンショット 2022-08-05 17 30 57" src="https://user-images.githubusercontent.com/874280/183036727-e8170837-3928-47cc-9510-77bf4f785ba4.png">